.github/workflows/test_production_function.yaml

```diff
@@ -36,6 +43,7 @@ jobs:
         id: generate-matrix
         run: |
           $params = $env:TestParams
+          echo $params.Length
```

Comment:
Style (see https://github.com/PoshCode/PowerShellPracticeAndStyle/blob/d1c6ae146b1060ec4844afc393161c6237bd85a7/Style-Guide/Naming-Conventions.md#naming-conventions)
`echo` is an alias for `Write-Output`